3.Arrays-and-Loops/descend.js

```diff
@@ -0,0 +1,4 @@
+let array = [1,6,7,8,3,4,5,6]
+array.sort()//sorting in ascending order

```

Comment:
try to do this with array [1,60,12, 13,7,8,3,4,5,6] and you get [1,12,13,3,4,5,6,60,7]. 
Read documentation about sort https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort